sqlmesh/core/model/definition.py

```diff
@@ -1823,6 +1844,12 @@ def _data_hash_values_no_sql(self) -> t.List[str]:
         for column_name, column_hash in self.column_hashes.items():
             data.append(column_name)
             data.append(column_hash)
+
+        # Include grants in data hash for seed models to force recreation on grant changes
+        # since seed models don't support migration
+        data.append(json.dumps(self.grants, sort_keys=True) if self.grants else "")
```

Comment:
Should we do this only if grans are non empty? Otherwise this is a breaking change that requires migration.

Reply:
ah, I found the migration script